frontend/src/app/home/map/map.component.ts

```diff
@@ -188,14 +188,13 @@ export class MapComponent implements OnInit {
 
                     switch (customType) {
                       case 'user':
-                        this.snackService.showMessage(`Dear Soleil00 You"re located here`);
+                        this.snackService.showMessage(`You are located here`);
```

Comment:
mental note here: this is not yet translated

Reply:
Yeah, I'm aware.  WIP.  🙃